Tests/Cosmos.Compiler.Tests.Bcl/System/GuidTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Sys = System;
```

Comment:
This isn't needed. There's no Guid type in Cosmos.System, so it can simply be `using System;`.